independent-projects/tools/devtools-common/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     if: ${{github.event.pull_request.merged == true}}
 
     steps:
-      - uses: radcortez/project-metadata-action@master
+      - uses: radcortez/project-metadata-action@1.0
```

Comment:
I would move to main. Better that changing all the workflow files in all the Quarkiverse if there is a fixed issue.

Reply:
That's not the best practice according GitHub actions: https://docs.github.com/en/actions/creating-actions/about-actions#using-release-management-for-actions

Ideally it should be `v1` and then it doesn't matter how many 1.x releases would happen, since they are all (supposed to be) compatible